### DISCUSSION TITLE
Canon RF 24-70mm f/2.8L IS USM

### DISCUSSION THREAD

**Discussion by nschaer** [lensfun/lensfun#1922] [Q&A] [answered]
Hi
I have the objectiv Canon RF 24-70mm f/2.8L IS USM. This isn't in the lensfun database. Could someone help me?
Thanks :-)
Kind regards
Nathalie

**Comment by tuxfanx** [comment, marked as answer]
A few calibration pictures are required for a calibration with the lensfun tools. You can carry out the calibration yourself and then provide us with your correction data, or you provide us with the pictures you have taken and we use them to generate the correction data for the db. You can read everything you need to know [here](https://pixls.us/articles/create-lens-calibration-data-for-lensfun/) and [here](https://github.com/lensfun/lensfun/blob/master/docs/calibration_tutorial/lens_calibration_tutorial.rst).
